LibreHardwareMonitorLib/Hardware/ISensor.cs

```diff
@@ -123,4 +123,9 @@ public interface ISensor : IElement
     /// Resets a value stored in <see cref="Max"/>.
     /// </summary>
     void ResetMax();
+
+    /// <summary>
+    /// Resets the plot graph.
+    /// </summary>
+    void ResetPlot();
```

Comment:
ResetPlot > ClearValues. Also update comment.

Reply:
Not entirely sure I understand what you're asking. Do you want the function to be called ClearValues instead of ResetPlot? And update the comment to what?

Is this what you're asking for

```
/// <summary>
/// Clears the values stored in <see cref="_values"/>.
/// </summary>
void ClearValues();
```

Reply:
Yes, that's right! The comment should refer to https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/fcf86b3c9e5943c17ea7c23cb1e3f244ff7eb2c7/LibreHardwareMonitorLib/Hardware/ISensor.cs#L113 though.